packages/valory/agents/oracle/aea-config.yaml

```diff
@@ -110,6 +110,7 @@ models:
       observation_interval: ${SKILL_ORACLE_ABCI_MODELS_PARAMS_ARGS_OBSERVATION_INTERVAL:int:300}
       max_healthcheck: ${SKILL_ORACLE_ABCI_MODELS_PARAMS_ARGS_MAX_HEALTHCHECK:int:1200}
       broadcast_to_server: ${SKILL_ORACLE_ABCI_MODELS_PARAMS_ARGS_BROADCAST_TO_SERVER:bool:false}
+      validate_timeout: ${SKILL_ORACLE_ABCI_MODELS_PARAMS_ARGS_VALIDATE_TIMEOUT:int:1205}
```

Comment:
is 1205 the previous default?

Reply:
Yes, this is the value in the skill's config. There was no override before on the agent or service level.